### DISCUSSION TITLE
修改出块时间后启动链，链跑到第299个区块就会卡住，提示”unexpected epoch change"

### DISCUSSION THREAD

**Discussion by peilinnnnnn** [ParityAsia/substrate-newsletter#20] [Substrate/Polkadot 技术问题] [answered]
如题，我使用paritytech/substrate v2.0.1 启动链，默认出块时间为3秒，我想设置为6秒，所以就将substrate\bin\node\runtime\src\constants.rs 里的 MILLISECS_PER_BLOCK: Moment = 3000改为了6000。最终出块时间虽然成功变成了6秒，但每次链跑到第299个区块就会卡死，提示”unexpected epoch change"。请问老师这个问题该怎么解决呢？
![image](https://user-images.githubusercontent.com/77475610/110207034-2e429900-7ebc-11eb-9d93-79ad71b5d16c.png)

**Comment by kaichaosun** [comment, marked as answer]
我用substrate node v2.0.1 的dev模式，没遇到这个情况。
你如果是多验证人节点，确保你的验证人没有下线，并且用的 chain spec 是符合 setup private network 教程里的。

**Reply by peilinnnnnn** [reply]
节点应该是没有问题，因为我启动了很多次，都是卡在第299个区块，我查查chain spec吧，感谢老师指点。

**Reply by peilinnnnnn** [reply]
这个问题终于解决了，原来是因为改了时间后需要至少使用spec.json默认的至少4个验证人。但我之前图省事，删掉了一个，虽然之前我3秒出块的时候只用3个验证人没有问题。但修改为6秒出块就需要恢复到系统默认的至少4个验证人了。我现在改为4个，就没有问题了。感谢老师解答。
